packages/nocodb/src/modules/jobs/jobs/webhook-handler/webhook-handler.processor.ts

```diff
@@ -0,0 +1,31 @@
+import { Process, Processor } from '@nestjs/bull';
+import { forwardRef, Inject, Logger } from '@nestjs/common';
+import { Job } from 'bull';
+import { JOBS_QUEUE, JobTypes } from '~/interface/Jobs';
+import { HookHandlerService } from '~/services/hook-handler.service';
+
+@Processor(JOBS_QUEUE)
+export class WebhookHandlerProcessor {
+  private logger = new Logger(WebhookHandlerProcessor.name);
+
+  constructor(
+    @Inject(forwardRef(() => HookHandlerService))
+    private readonly hookHandlerService: HookHandlerService,
+  ) {}
+
+  @Process(JobTypes.HandleWebhook)
+  async job(job: Job) {
+    /*
+      job.data: {
+        hookName: string;
+        prevData;
+        newData;
+        user: UserType;
+        viewId: string;
+        modelId: string;
+        tnPath: string;
+      }
+    */
```

Comment:
You can define the type and pass as generic type

```
job: Job<JobDataType>
```

Reply:
Done